docs/community/KPI-options.md

```diff
@@ -0,0 +1,41 @@
+---
+title: KPI Options
+sidebar_label: KPI Options
+---
+## KPI Option Overview
+
+Community options, growth options or performance options - the naming is not the point, more that these options are designed to create aligned incentives with your community to grow. We have decided to call them *Key Performance Indicator options* or *KPI options*. 
+
+### What are these options? 
+
+Using UMA’s Expiring Multi Party (EMP) contract template, any project can create an incentive mechanism which offers rewards to their targeted community in exchange for the community’s support in helping them achieve their Key Performance Metrics. 
+
+At a high-level, KPI options are synthetic ERC20 tokens that can be redeemed , but only if the project hits it’s predetermined KPI targets before expiry. These targets are up to each project and could be TVL, number of contracts deployed, number of transactions, etc.
+
+Every community member airdropped these KPI options, becoming vested in helping the project succeed on its Key Metrics, as the token grows more valuable as the target approaches.  This provides powerful incentive alignment between the protocol and its community. 
+
+### How it works?
+
+The first step is create an UMIP to add your token to our list of accepted collateral currencies (if it isn’t already). Then you'll need to 
```

Comment:
To avoid duplication, lets refer them to https://docs.umaproject.org/uma-tokenholders/umips to understand the UMIP process